### PR TITLE
chore(i18n): update translation

### PR DESCRIPTION
**Pull request by Ende93**

### CODE REVIEW

**Comment by Ende93** [comment]
add `content/zh/guide/v10/differences-to-react.md`

File: content/zh/guide/v10/differences-to-react.md [hunk 1]
```diff
@@ -0,0 +1,220 @@
+---
+name: 与 React 的差别
+permalink: '/guide/differences-to-react'
+description: '本文详细说明了 Preact 和 React 之间的差别'
+---
+
+# 与 React 的差别
+
+Preact 本身并没有去重新实现一遍 React。它们之间有些不同之处。但大部份差别都是很细微的，而且可以通过一个轻量的 [preact-compat] Preact 层实现 100% 兼容 React。
```
Comment (rschristian) [inline content/zh/guide/v10/differences-to-react.md:9]:
```suggestion
Preact 本身并没有去重新实现一遍 React。它们之间有些不同之处。但大部份差别都是很细微的，而且可以通过一个轻量的 [preact/compat] Preact 层实现 100% 兼容 React。
```

File: content/zh/guide/v10/differences-to-react.md [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+- 事件不会冒泡到 `<Portal>` 组件
+- 在表单输入中时应当使用标准的 `onInput` 来代替 React 的 `onChange` (**仅当不使用 `preact/compat` 的时候**)
+- 应当用 `onDblClick` 来代替 React 的 `onDoubleClick` (**仅当不使用 `preact/compat` 的时候**)
+- `<input type="search">` 应当使用 `onSearch`，因为在 IE11 上 "x" 按钮并不支持 `onInput` 
+
+另一个显著的区别是 Preact 更严格地遵循 DOM 规范。支持像任何其他元素一样的自定义元素，并且支持区分大小写的自定义事件名称（就像它们在 DOM 中一样）。
+
+## 版本兼容
+
+对于 preact 和 [preact-compat]， 版本兼容通过_当前_和_之前_的 React 主要版本去衡量。当 React 团队公布新的特性的时候，若符合 [Project Goals] 它们可能会被添加到 Preact 的核心代码当中。这是一个相对民主的迭代过程，我们会持续使用 issues 和 pull request 来公开、持续进行讨论和决策。
```
Comment (rschristian) [inline content/zh/guide/v10/differences-to-react.md:34]:
```suggestion
对于 preact 和 [preact/compat]， 版本兼容通过_当前_和_之前_的 React 主要版本去衡量。当 React 团队公布新的特性的时候，若符合 [Project Goals] 它们可能会被添加到 Preact 的核心代码当中。这是一个相对民主的迭代过程，我们会持续使用 issues 和 pull request 来公开、持续进行讨论和决策。
```

**Comment by rschristian** [comment]
Thanks! Sorry for the delay, a lot of us just had holidays.

**Review by rschristian — APPROVED** [review]

**Pull request merged**